dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -1418,8 +1418,8 @@ components:
             - SWL
             - NFL
             - service
-            
-            
+            - cwl
```

Comment:
refresh my memory, why does this keep on changing?

Reply:
🤷